### PR TITLE
Implement io.ReadSeeker interface for reflectedStream

### PR DESCRIPTION
**Pull request by sayplastic**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,4 +39,4 @@ clean:
 
 .PHONY: server
 server:
-	go run . serve
+	LW_DEBUG=1 go run . serve
```
Comment (lyoshenka) [inline Makefile]: careful committing this, and all the other debug stuff

File: player/player.go [hunk 2]
```diff
@@ -48,47 +50,76 @@ func PlayURI(uri string, rangeHeader string, w http.ResponseWriter) (err error)
...
+	var zero, newSeekOffset int64
+
+	if whence == io.SeekEnd {
+		newSeekOffset = s.Size - 1 - offset
+	} else if whence == io.SeekStart {
+		newSeekOffset = offset
+	} else if whence == io.SeekCurrent {
+		newSeekOffset = s.seekOffset + offset
+	} else {
+		return zero, e.New("invalid seek whence argument")
```
Comment (lyoshenka) [inline player/player.go]: why do you need a `zero` var? why not just return `0`?

File: Makefile (hunk 1 above)
Comment (sayplastic) [reply Makefile]: makefile is only for development, where debug mode by default makes sense

File: player/player.go (hunk 2 above)
Comment (sayplastic) [reply player/player.go]: I think it was a brain fart

**Comment by sayplastic** [comment]
@lyoshenka I decided to man up and copy/paste/patch streaming http method from `net/http` to make my implementation work. Turns out non-streaming formats delivery also works fine when ranges are implemented correctly. Could you please give it a cursory glance?

**Pull request merged**